content/en/docs/setup/production-environment/tools/kubeadm/enable-disable-dual-stack.md

```diff
@@ -0,0 +1,147 @@
+---
+reviewers:
+- sig-cluster-lifecycle
+- neolit123
+- sftim
+title: Enable Or Disable Dual-Stack via kubeadm
+feature:
+  title: Enable Or Disable Dual-Stack via kubeadm
+content_type: task
+weight: 110
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.21" state="beta" >}}
+
+IPv4/IPv6 dual-stack enables the allocation of both IPv4 and IPv6 addresses to {{< glossary_tooltip text="Pods" term_id="pod" >}} and {{< glossary_tooltip text="Services" term_id="service" >}}.
+
+<!-- body -->
+
+## Enable dual-stack via kubeadm
+
+Install kubeadm following the steps from the [Installing Kubeadm](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/) documentation.
+
+Make sure that nodes allow IPv6 forwarding, if not, run `sudo sysctl -w net.ipv6.conf.all.forwarding=1` on every node in the cluster.
+
+{{< note >}}
+`kubeadm upgrade` will change `IPv6DualStack` to true by default if feature gate is not set in old cluster. However, cluster-cidr and service-cidr modification are not supported.
+{{< /note >}}
+
+### Create a dual-stack cluster
+
+A simple command with `podCidr` and `serviceCidr` flags to create a dual-stack cluster via `kubeadm init`, it like below:
+
+```shell
+kubeadm init --feature-gates IPv6DualStack=true --pod-network-cidr=10.244.0.0/16,fd00:10:244::/56 --service-cidr=10.96.0.0/16,fd00:10:96::/112
```

Comment:
```suggestion
kubeadm init --pod-network-cidr=10.244.0.0/16,fd00:10:244::/56 --service-cidr=10.96.0.0/16,fd00:10:96::/112
```

since the FG becomes beta in 1.21 we don't need to pass `--feature-gates IPv6DualStack=true`